bucket/clash.meta.json

```diff
@@ -0,0 +1,35 @@
+{
+    "version": "1.11.2",
+    "description": "Fork of Clash, a rule-based tunnel.",
+    "homepage": "https://github.com/MetaCubeX/Clash.Meta",
+    "license": "GPL-3.0-or-later",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/MetaCubeX/Clash.Meta/releases/download/v1.11.2/Clash.Meta-windows-amd64-v1.11.2.zip",
+            "hash": "9d5013f536011adf573a51a8bec21e021f8ff0964713bf403b1cb4751616379c"
+        },
+        "32bit": {
+            "url": "https://github.com/MetaCubeX/Clash.Meta/releases/download/v1.11.2/Clash.Meta-windows-386-v1.11.2.zip",
+            "hash": "396a7d2295e93700eca433f277b52eb82e797a3426d57b183d6ceb40b9ae66aa"
+        }
+    },
+    "pre_install": "Get-ChildItem \"$dir\\Clash.Meta*.exe\" | Rename-Item -NewName 'Clash.Meta.exe'",
+    "bin": [
+        "Clash.Meta.exe",
+        [
+            "Clash.Meta.exe",
+            "clash"
```

Comment:
Wouldn't this conflict with main/clash's binaries?

Reply:
Yes, but my thought is,
Since *Clash.Meta* is a fork of *Clash*:

(1) It should be a rare case for user to install both of them. Even if so, they can run `scoop reset clash` to solve the problem. (so that the shim `clash` points to *Clash*, and `clash.meta` points to *Clash.Meta*)

(2) Users will expect the shim `clash` to work for *Clash.Meta* as well.

Reply:
Hmm, understood